lib/auth/touchid/authenticate.h

```diff
@@ -0,0 +1,18 @@
+#ifndef AUTHENTICATE_H_
```

Comment:
Shouldn't we have `Copyright` in all files?

Reply:
That's a good question. `make fix-license` basically only cares about Go, but I can add headers to the files we have here, since we know for sure they are new.